style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 60px 10%;
+}
+
+nav a {
+  text-decoration: none;
+  font-size: 30px;
+  color: #6070ff;
+  font-weight: 700;
+}
+
+.content {
+  flex-basis: fit-content;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
+  padding: 120px 10%;
```

Comment:
I kindly suggest following the previous reviewer's advice and using flexbox in this section, this will help you center the elements on the screen and fit any mobile device.

Reply:
Hi @mgmediaweb 

I guess i'm missing the properties display and flex direction then. 

Thanks a lot for your review.